wgsl/index.bs

```diff
@@ -5125,19 +5125,14 @@ single texel is returned.
 Reads a single texel from a texture without sampling or filtering.
 
 ```rust
-textureLoad(t : texture_1d<T>, coords : i32) -> vec4<T>
-textureLoad(t : texture_1d_array<T>, coords : i32, array_index : i32) -> vec4<T>
-textureLoad(t : texture_2d<T>, coords : vec2<i32>) -> vec4<T>
+textureLoad(t : texture_1d<T>, coords : i32, level : i32) -> vec4<T>
```

Comment:
MSL:

> The following member functions can perform sampler-less reads from a 1D texture. Since
mipmaps are not supported for 1D textures, lod must be 0.

Reply:
Yes, we don't support mipmaps on 1D textures in WebGPU.
I was just thinking that making this consistent between the sampled texture types would be beneficial, and if we ever decide to allow mipmaps on 1D textures, e.g. via an extension, the WGSL will not need to be changed.

Reply:
I'm on the fence about this - it certainly adds some ugly special casing in the compiler - made even worse when said extension is introduced.
The confusion around the `level` parameter would be less so if we had put these into a `textureLoadLevel` and commented that `textureLoad` only operates on the first-mip level. Then the signatures would have also mirrored `textureStore` and `textureSample`.
If we go with this, please can we have a note in the spec to say `level` for 1D textures must be 0 or is just ignored?